data/campaigns/The_Rise_Of_Wesnoth/scenarios/12_A_Final_Spring.cfg

```diff
@@ -40,21 +39,121 @@
         name= _ "King Addroran IX"
         profile="portraits/addroran.webp"
         canrecruit=yes
-        {GOLD 200 160 120}
+        gold=200 # extra gold to make up for him not starting with loyal swordsmen (which he did in the previous version)
+        income=13
         team_name=Haldric
         user_team_name=_"Humans"
-        recruit=Bowman, Horseman, Mage, Spearman, Knight, Swordsman, Peasant
+        recruit=Heavy Infantryman,Mage,Javelineer,Swordsman
         [ai]
             {NO_SCOUTS}
-            grouping=defensive
-            passive_leader=yes
-            recruitment_pattern=fighter,fighter,archer,mixed fighter
         [/ai]
         {FLAG_VARIANT loyalist}
     [/side]
-
     {STARTING_VILLAGES 2 10}
 
+    # allow King Addroran to reach the oasis hidden under the nearby ship, since he's likely to get poisoned at some point
+    {SILENTLY_LIMIT_LEADER_MOVES 2 1}
+    [event]
+        name=prestart
+        [object]
+            [filter]
+                id=King Addroran IX
+            [/filter]
+            [effect]
+                apply_to,replace=movement_costs,yes
```

Comment:
Here's another one. Is that fine?

Reply:
yeah I think these should go on separate lines, too